plugins/woocommerce-admin/client/marketplace/components/header/header.scss

```diff
@@ -9,6 +9,9 @@
 		"mktpl-tabs mktpl-tabs mktpl-tabs" auto / 1fr 320px 36px;
 	padding: 0 $content-spacing-large;
 	width: 100%;
+	position: sticky;
+	top: var(--wp-admin--admin-bar--height, 32px);
+	z-index: 1;
```

Comment:
I chose the minimum possible z-index here, although I can understand an argument for setting it to 10 or 100.